### DISCUSSION TITLE
[Solid] Need help making the Select's content take the width of the trigger

### DISCUSSION THREAD

**Discussion by stanlrt** [chakra-ui/ark#2668] [Q&A] [answered]
Hello,

I have tried achieving what's described in the title using multiple ways (removing the Portal, setting position to relative, setting width and min-width to 100%...) but I always end up with the content either taking the width of its children (the items), or filling 100% of the page's width.
I tried reproducing what I see when I inspect the docs, to no avail

Is there a repo/example somewhere showing the styling?

**Comment by elson-currentcorp** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/e24a394a-c06d-46b4-8e73-73e522e00861)
Have you tried setting sameWidth to true for the positioning prop?

**Reply by stanlrt** [reply]
Thanks, didn't know about it. Sadly it does not work. Does the `asChild` make a difference?

**Reply by stanlrt** [reply]
Here the Select code for reference: https://gist.github.com/stanlrt/e355e9ffcf8311088ed70df618ec512b
If needed I can create a full MRE later

**Comment by stanlrt** [comment]
Well nevermind, after my laptop died the dev server was restarted... and it works
